docs/equipment/index.md

```diff
@@ -0,0 +1,33 @@
+# Tools & Equipment
+All tools have risks. If you see an unsafe tool, or a tool being used unsafely, tell the person to stop using the tool and explain why if you are able. We’re all learning in Hackspace, and it’s important you are friendly and respectful in your manner.
+
+Our tools fall into a few different categories:
+* Immediately useable
+* Induction protected
+* No lone working
+* No Entry
+* Not ready for use
+* Not permitted in the space
+
+The majority of our tools are available for immediate use, This includes all non-powered tools and a few powertools.
+
+* Do not use any tool you’re not comfortable using. Many of our members find Youtube or our books in the general space a great resource.
+* Wear PPE - we provide basic PPE, but it is your responsibility to protect yourself and others. See [Health & Safety section](https://docs.hacman.org.uk/governance/health_safety/health_and_safety)
+* Tools have [risk assessments](https://docs.hacman.org.uk/governance/health_safety/risk_assessments) -  please read these before using any tool.
+
+## Inductions
+Certain equipment requires an induction so you don't hurt yourself or damage the machines. For example: the Laser Cutter, 3D printers, sewing machines, lathes, etc. 
+
+Please check signage around the machine. A good general rule to go by is that if it's particularly dangerous or complex, it's probably induction protected.
+
+The “Tools & Equipment” page on the [Members Portal](https://members.hacman.org.uk/equipment) notes which pieces of equipment require inductions.
+
+### No Lone Working
+Some induction protected tools are additionally marked ‘No Lone Working’. This means another member must be in the space. They do not have to be in the same room, but need to be able to hear you in the case of an emergency.
+
+If you have been asked to listen out for someone, please let them know when you are planning on leaving the space.
+
+For more information see [Health & Safety section](https://docs.hacman.org.uk/governance/health_safety/health_and_safety) 
+
+### Not Ready for Use
+Some tools will be marked as not ready for use. This is usually because they have not had their risk assessments/training program completed. Sometimes it is because we don’t have the right infrastructure or ventilation installed yet. We are always looking for members willing to help with all these processes.
```

Comment:
I don't think we have signage on these either to be fair. Will put on Trello for us to do.